src/server.py

```diff
@@ -361,6 +361,14 @@ def finished(self):
 
                 output_logger.close()
 
+                # What's the diffrence between "get_config()" and "config" ?
```

Comment:
There is no difference. 
I've created a getter method because I was used to it in Java. However in python it's not needed and you can safely access class fields directly. So, please use "config.output_files"